Description: Align xstatic imports with horizon package.
  xstatic packages are vendored into horizon via the
  python3-django-horizon package and installed in
  /usr/lib/python3/dist-packages/horizon/xstatic.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: not-needed

--- a/heat_dashboard/enabled/_1650_project_template_generator_panel.py
+++ b/heat_dashboard/enabled/_1650_project_template_generator_panel.py
@@ -38,18 +38,18 @@
 JS_BASE = '%s/js' % TEMPLATE_GENERATOR_BASE
 
 ADD_XSTATIC_MODULES = [
-    ('xstatic.pkg.angular', [
+    ('horizon.xstatic.pkg.angular', [
         'angular-animate.js',
         'angular-aria.js',
         'angular-messages.js',
     ]),
-    ('xstatic.pkg.angular_bootstrap', ['angular-bootstrap.js']),
+    ('horizon.xstatic.pkg.angular_bootstrap', ['angular-bootstrap.js']),
     # ('xstatic.pkg.angular_material', ['angular-material.js']),
-    ('xstatic.pkg.angular_uuid', ['angular-uuid.js']),
-    ('xstatic.pkg.angular_vis', ['angular-vis.js']),
-    ('xstatic.pkg.filesaver', ['FileSaver.js']),
-    ('xstatic.pkg.js_yaml', ['js-yaml.js']),
-    ('xstatic.pkg.json2yaml', ['json2yaml.js']),
+    ('heat_dashboard.xstatic.pkg.angular_uuid', ['angular-uuid.js']),
+    ('heat_dashboard.xstatic.pkg.angular_vis', ['angular-vis.js']),
+    ('heat_dashboard.xstatic.pkg.filesaver', ['FileSaver.js']),
+    ('heat_dashboard.xstatic.pkg.js_yaml', ['js-yaml.js']),
+    ('heat_dashboard.xstatic.pkg.json2yaml', ['json2yaml.js']),
 ]
 
 ADD_SCSS_FILES = [
--- a/requirements.txt
+++ b/requirements.txt
@@ -13,9 +13,3 @@
 # This will be installed from git in OpenStack CI if the job setting
 # required-projects for horizon:
 horizon>=18.3.1 # Apache-2.0
-
-xstatic-angular-uuid>=0.0.4.0 # MIT
-xstatic-angular-vis>=4.16.0.0 # MIT
-xstatic-filesaver>=1.3.2.0 # MIT
-xstatic-js-yaml>=3.8.1.0 # MIT
-xstatic-json2yaml>=0.1.1.0 # MIT
